Makefile

```diff
@@ -50,7 +58,7 @@ test: start  ## Run tests
 	fi
 
 .PHONY: lint
-lint: 
+lint:
 	docker-compose run --no-deps --rm web /bin/bash -c 'flake8; mypy app --config="../mypy.ini"'
```

Comment:
Do we want to switch this to invoke pre-commit for consistency?

Reply:
Done.